sass/bourbon/css3/_flex-box.scss

```diff
@@ -1,20 +1,5 @@
 // CSS3 Flexible Box Model and property defaults
 
-// Custom shorthand notation for flexbox
-@mixin box($orient: inline-axis, $pack: start, $align: stretch) {
-  @include display-box;
-  @include box-orient($orient);
-  @include box-pack($pack);
-  @include box-align($align);
-}
-
-@mixin display-box {
```

Comment:
This `display-box` mixin is only used in the `box` mixin on line 4 of this file. The `box` mixin in-turn does not appear to be used anywhere in the project. I decided to delete these two mixins rather than maintain them. Also, it does not look like they can be used outside of the project.